packages/core/src/MatchMaker.ts

```diff
@@ -346,16 +350,15 @@ export async function handleCreateRoom(roomName: string, clientOptions: ClientOp
   room.listing = driver.createInstance({
     name: roomName,
     processId,
-    ...additionalListingData
+    ...additionalListingData,
   });
 
   if (room.onCreate) {
     try {
       await room.onCreate(merge({}, clientOptions, registeredHandler.options));
 
       // increment amount of rooms this process is handling
-      presence.hincrby(getRoomCountKey(), processId, 1);
-
+      await presence.hincrby(getRoomCountKey(), processId, 1);
```

Comment:
it is not important to await this unless we want to catch the error.  Of course, this might be a nice one to catch, as if REDIS is dying we could save having an unhandled exception, but using `.catch()` could be more performant.